Library/Formula/mapext.rb

```diff
@@ -0,0 +1,18 @@
+class Mapext < Formula
+  desc "map files from one extension to another via a specified command"
+  homepage "http://www.acme.com/software/mapext/"
+  url "http://www.acme.com/software/mapext/mapext_10nov1995.tar.gz"
```

Comment:
Does this mean there hasn't been a release in 20 years? We don't usually add abandoned software unless it's very widely used.